test/BUILD.gn

```diff
@@ -118,6 +118,7 @@ test("brave_unit_tests") {
   if (brave_rewards_enabled) {
     sources += [
       "//brave/vendor/bat-native-ledger/src/bat/ledger/internal/media/helper_unittest.cc",
+        "//brave/vendor/bat-native-ledger/src/bat/ledger/internal/media/reddit_unittest.cc",
```

Comment:
Nit: Sometimes hard to tell in a diff, but looks like extra spacing at the beginning of this line.

Reply:
There was. Removed.